added a new flag to github gitops (#172)
port-labs/port-docs: docs/api-providers/gitops/github-app-gitops/advanced-configuration.md

| `defaultTeam` | `String` | Name of the default team to use, if one is not provided in the `port.yml` file | `null`
| `defaultPropertiesPath` | `String` | Changes the `properties` location in the `port.yml` file, you can provide custom paths like `props`, `data` or any other key. (in order to make it a top-level key you have to provide `.`).<br/> A common use case is to reuse existing `yml` files within the repository without the need to change it to the `port.yml` default format | `properties`
| `deleteDependentEntities` | `Boolean` | Flag to enable deletion of dependent Entities - an Entity that has a required Relation with an Entity you wish to delete | `false`

> | `enableMergeEntity` | `Boolean` | Flag to enable merging an entity instead of overriding the current contents of it. <br/> More info about merging entities can be found at [The Entity Tutorial](https://docs.getport.io/software-catalog/entity/tutorial/#post-request) | `false`

Reviewer: ```suggestion
| `enableMergeEntity` | `Boolean` | Flag to enable merging new values with the existing Entity values, instead of overriding all existing values. <br/> More info about merging entities can be found in the [Entity Tutorial](https://docs.getport.io/software-catalog/entity/tutorial/#post-request) | `false`
```
